src/Microsoft.ML.Experimental/MLContextExtensions.cs

```diff
@@ -11,5 +14,105 @@ public static class MLContextExtensions
         /// </summary>
         /// <param name="ctx"><see cref="MLContext"/> reference.</param>
         public static void CancelExecution(this MLContext ctx) => ctx.CancelExecution();
+
+        /// <summary>
+        /// Normalize (rescale) the column according the <see cref="NormalizingEstimator.NormalizationMode.MinMax"/> mode.
+        /// It normalizes the data based on the observed minimum and maximum values of the data.
+        /// </summary>
+        /// <param name="catalog">The transform catalog</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
+        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="ensureZeroUntouched">Whether to map zero to zero, preserving sparsity.</param>
```

Comment:
>ensureZeroUntouched [](start = 25, length = 19)

Do we like this "untouched?" I mean, I guess it's fine, but...

Reply:
updated to use "fixZero" in the API

internally, our codebase uses a mix of 'fixZero' and 'ensureZeroUntouched' , will create a separate bug to clean this up  post  v1

---
In reply to: [268184652](https://github.com/dotnet/machinelearning/pull/3052#discussion_r268184652) [](ancestors = 268184652)